yaksh/views.py

```diff
@@ -954,3 +958,79 @@ def design_questionpaper(request):
         context = {'form': form, 'questionpaper':True}
         return my_render_to_response('yaksh/design_questionpaper.html',
                                      context, context_instance=ci)
+
+@login_required
+def search_teacher(request, course_id):
+    """ search teachers for the course """
+
+    user = request.user
+    ci = RequestContext(request)
+    context = {}
+    course = get_object_or_404(Course, creator=user, pk=course_id)
+    context['course'] = course
+    if not user.is_authenticated() or not is_moderator(user):
+        raise Http404('You are not allowed to view this page!')
+
+    if request.method == 'POST':
+        u_name = request.POST.get('uname')
+        if len(u_name) == 0:
+            context['success'] = False
+            return my_render_to_response('yaksh/addteacher.html', context,
+                                        context_instance=ci)
+        else:
+            teachers = User.objects.filter(Q(username__icontains=u_name)|
+                Q(first_name__icontains=u_name)|Q(last_name__icontains=u_name)|
+                Q(email__icontains=u_name)).exclude(is_superuser=1)
```

Comment:
is it possible to exclude the current user here itself instead of doing it in the template?